local/client/conn/conn.go

```diff
@@ -0,0 +1,66 @@
+package conn
+
+import (
+	"github.com/scootdev/scoot/local/protocol"
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+	"log"
+	"net"
+	"time"
+)
+
+// A Dialer can dial a connection to the Scoot server.
```

Comment:
I'm fine with whatever style we pick. I'd even lean towards "it's not worth enforcing a consistent style for this", but I'm happy to pick one and stick with it. WDYT?